toolchains/rbe_toolchains_config.bzl

```diff
@@ -36,7 +36,7 @@ _CLANG_LIBCXX_ENV = dicts.add(_CLANG_ENV, {
 _GCC_ENV = {
     "BAZEL_COMPILER": "gcc",
     "BAZEL_LINKLIBS": "-l%:libstdc++.a",
-    "BAZEL_LINKOPTS": "-lm",
+    "BAZEL_LINKOPTS": "-lm:-fuse-ld=gold",
```

Comment:
this seems obsolete - it didnt seem to make any difference when regenerating chains